examples/twittermap/noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/FileFeedDriver.java

```diff
@@ -83,10 +115,17 @@ private void doMain(String[] args) {
         } finally {
             try {
                 br.close();
+                elapsedTime = (System.currentTimeMillis() - startTime) / 1000;
```

Comment:
same code as line 98 to 102? maybe consider to wrap up as a separate function?

Reply:
Sure.